source/addComment.js

```diff
@@ -1,3 +1,24 @@
 $(document).ready(function(){
-  // add your code here
-})
+  $('#new_comment_button').on('click', function () {
+       event.preventDefault();
+
+     $('#new_comment_button').hide()
+     $( '<form id="new_comment" name="new_comment" style="display: block"></form>' ).appendTo( "body" );
+     $('<textarea form="new_comment" id="comment-text" placeholder="your comment goes here"/>').appendTo("#new_comment");
+     $('<input type="text" id="authorName" placeholder="your name"/>').appendTo("#new_comment");
+     $('<input type="submit" id="createComment" value="Create Comment!"/>').appendTo("#new_comment");
```

Comment:
Think about how you could use `hide` and `show` to replace `append` and `remove`...